doc/nrf/ug_matter_overview_device_types.rst

```diff
@@ -0,0 +1,210 @@
+
+.. _ug_matter_device_types_sensors:
+
+Sensor device types
+===================
+
++-------------------+------------------------------------------------------------------------------+---------------------+---------------------------------------+
+| Device type       | Description (from Device Library specification)                              | State of the device | Dedicated sample in the |NCS|         |
++===================+==============================================================================+=====================+=======================================+
+| Contact Sensor    | A Contact Sensor device reports boolean state (open/close                    | Certifiable         |                                       |
+|                   | or contact/no-contact).                                                      |                     |                                       |
++-------------------+------------------------------------------------------------------------------+---------------------+---------------------------------------+
+| Light Sensor      | A Light Sensor device is a measurement and sensing device that is capable    | Certifiable         |                                       |
+|                   | of measuring and reporting the intensity of light being emitted              |                     |                                       |
+|                   | by a light source.                                                           |                     |                                       |
++-------------------+------------------------------------------------------------------------------+---------------------+---------------------------------------+
+| Occupancy Sensor  | An Occupancy Sensor is a measurement and sensing device that is capable      | Certifiable         |                                       |
+|                   | of measuring and reporting the occupancy state in a designated area.         |                     |                                       |
++-------------------+------------------------------------------------------------------------------+---------------------+---------------------------------------+
+| Temperature       | A Temperature Sensor device reports measurements of temperature.             | Certifiable         | :ref:`matter_weather_station_app`     |
+| Sensor            |                                                                              |                     |                                       |
++-------------------+------------------------------------------------------------------------------+---------------------+---------------------------------------+
+| Pressure Sensor   | A Pressure Sensor device measures and periodically reports the pressure      | Provisional         | :ref:`matter_weather_station_app`     |
```

Comment:
Based on current spec the Pressure Sensor is not provisional

Reply:
Changed.